erpnext/crm/doctype/opportunity/opportunity.js

```diff
@@ -122,10 +122,37 @@ frappe.ui.form.on("Opportunity", {
 		}
 	},
 
+	currency: function(frm) {
```

Comment:
On changing the currency, labels for all currency fields should show the currency too.

![image](https://user-images.githubusercontent.com/24353136/127468263-de356bf4-6d98-4f93-914d-d4c20d6cd1fc.png)